README.md

```diff
@@ -18,7 +18,27 @@
 - .NET 5.0 ([download](https://docs.microsoft.com/en-us/dotnet/core/install/))
 - Visual Studio Code (optional) ([download](https://code.visualstudio.com/download))
 
-## Running the System via Codespaces
+## Running loderunner via Codespaces
+
+1. Open codespaces in <https://github.com/retaildevcrews/loderunner>
+2. Configure CosmosDB Secrets for local use
+   - [Using Cosmos DB Emulator](src/LodeRunner.Data/README.md#using-cosmos-db-emulator)
+   - [Using shared Cosmos DB](src/LodeRunner.Data/README.md#using-shared-cosmos-db)
+3. Allow access to CosmosDB through firewall ([Instructions](src/LodeRunner.Data/README.md#cosmosdb-firewall-ip-ranges))
+   > NOTE: Skip this step if using Cosmos DB Emulator.
+4. Run LodeRunner.API
+
+   ```bash
+      cd src/LodeRunner.API
```

Comment:
this isn't needed, we should reuse LR.API local pod

Reply:
updated instructions to embed them with existing